.github/workflows/clean_issues.yml

```diff
@@ -0,0 +1,23 @@
+name: Close incomplete issues
+on:
+  schedule:
+    - cron: "30 1 * * *"
+
+jobs:
+  close-issues:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
```

Comment:
I think we can drop this permission since we aren't applying the PR-closing behaviour here.

Reply:
fixed! @tewinget also fixed the ngtcp2 bump